### PR TITLE
fix lowercase

### LINKED ISSUE #2412: Lowercase 'cockatrice' in mac menus
<b>OS:</b>
macOS Sierra 10.12

<b>Cockatrice version:</b>
2017-01-30
___
<br>

`cockatrice` lowercase in mac menu.
macmenu

### PR DESCRIPTION
**Pull request by ZeldaZach**
Fix #2412

### CODE REVIEW

File: cockatrice/CMakeLists.txt [hunk 1]
```diff
@@ -232,19 +232,19 @@ if(UNIX)
         set(MACOSX_BUNDLE_INFO_STRING "${PROJECT_NAME}")
         set(MACOSX_BUNDLE_GUI_IDENTIFIER "com.cockatrice.${PROJECT_NAME}")
         set(MACOSX_BUNDLE_LONG_VERSION_STRING "${PROJECT_NAME}-${PROJECT_VERSION}")
-        set(MACOSX_BUNDLE_BUNDLE_NAME ${PROJECT_NAME})
+        set(MACOSX_BUNDLE_BUNDLE_NAME "Cockatrice")
```
Comment (tooomm) [inline cockatrice/CMakeLists.txt]: Is this one needed for sure?

File: CMakeLists.txt [hunk 2]
```diff
@@ -212,21 +212,21 @@ option(WITH_SERVER "build servatrice" OFF)
...
-    SET(CPACK_INSTALL_CMAKE_PROJECTS "cockatrice;cockatrice;ALL;/" ${CPACK_INSTALL_CMAKE_PROJECTS})
+    SET(CPACK_INSTALL_CMAKE_PROJECTS "Cockatrice;Cockatrice;ALL;/" ${CPACK_INSTALL_CMAKE_PROJECTS})
 endif()
 
 # Compile oracle (default on)
 option(WITH_ORACLE "build oracle" ON)
 if(WITH_ORACLE)
     add_subdirectory(oracle)
-    SET(CPACK_INSTALL_CMAKE_PROJECTS "release/oracle.app;oracle;ALL;/" ${CPACK_INSTALL_CMAKE_PROJECTS})
+    SET(CPACK_INSTALL_CMAKE_PROJECTS "release/Oracle.app;Oracle;ALL;/" ${CPACK_INSTALL_CMAKE_PROJECTS})
```
Comment (Daenyth) [inline CMakeLists.txt]: I wonder why this is `Oracle.app` when the others are just `Cockatrice` etc ?

Comment (ctrlaltca) [inline CMakeLists.txt]: Nice catch @Daenyth; it can be changed safely

File: cockatrice/CMakeLists.txt (hunk 1 above)
Comment (ctrlaltca) [inline cockatrice/CMakeLists.txt]: It's not needed, better keep the old line

File: CMakeLists.txt (hunk 2 above)
Comment (ZeldaZach) [inline CMakeLists.txt]: Fixed

File: cockatrice/CMakeLists.txt (hunk 1 above)
Comment (ZeldaZach) [inline cockatrice/CMakeLists.txt]: Fixed

**Comment by ctrlaltca** [comment]
https://travis-ci.org/Cockatrice/Cockatrice/jobs/203998162#L1516
Errored out on linux, where the filesystem is case sensitive.
The desktop files must be renamed to have the first letter uppercase.

**Comment by ZeldaZach** [comment]
Changed the `*.desktop` back to lowercase.

**Pull request merged**